content/rc/subscriptions/_index.md

```diff
@@ -22,17 +22,14 @@ This page helps you manage your Redis Cloud subscriptions; it briefly compares a
 
 As of February 2021, Redis Enterprise Cloud supports the following subscription plans:
 
-- _Free plans_ are designed for training purposes and prototyping.  They can be seamlessly upgraded to Fixed plans with no data loss.  (Free plans are a tier of Fixed plans.)
+- [Free plans](#free-plans)
+- [Fixed plans](#fixed-plans)
+- [Flexible plans](#flexible-plans)
+- [Annual plans](#annual-plans)
 
-- _Fixed plans_ are cost-efficient and designed for low-throughput scenarios.  They support a range of availability, persistence, and backup options.  Pricing supports low throughput workloads.
+Here's a quick comparison between each plan:
 
-- _Flexible plans_ support more databases (and larger sizes), unlimited connections, and greater throughput.  Hosted in dedicated VPCs, they feature high-availability in a single or multi-AZ, active-active geo distribution, Auto Tiering, clustering, data persistence, and configurable backups.  Pricing is "pay as you go" to support any dataset size or throughput.
-
-- _Annual plans_ support the same features as Flexible plans, but at significant savings.  Annual plans also provide Premium support.  Further, the underlying commitment applies to all workloads across multiple providers and regions.
-
-Here's a quick comparison:
-
-| Feature | Free plan | Fixed plan | Flexible/<br/>Annual plan |
+| Feature | [Free plan](#free-plans) | [Fixed plan](#fixed-plans) | [Flexible](#flexible-plans)/<br/>[Annual](#annual-plans) plan |
```

Comment:
I'm not wild about the links in the table headers. There are already links above, links to the right, and the links themselves link to content that's just right below the table.